e2e/test/ccm_e2e_test.go

```diff
@@ -231,11 +231,17 @@ var _ = Describe("e2e tests", func() {
 	}
 
 	addNewNode := func() {
+		if useExisting {
```

Comment:
This doesn't look like something we want to have in our codebase? Maybe just conditionally skip the test instead?

Reply:
ya i can revert this and do a bigger PR where we drop all the tf stuff